ddtrace/tracer/span_test.go

```diff
@@ -77,7 +77,7 @@ func TestSpanFinish(t *testing.T) {
 	// the finish should set finished and the duration
 	time.Sleep(wait)
 	span.Finish()
-	assert.True(span.Duration > int64(wait))
+	assert.Greater(span.Duration, int64(wait))
```

Comment:
This test is flaky for windows. This change doesn't fix it but should help us identify what's going on. Windows has time issues.